regimes/my/tax_identity.go

```diff
@@ -0,0 +1,51 @@
+package my
+
+import (
+	"errors"
+	"regexp"
+	"strings"
+
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+var (
+	// Malaysian Business Registration Numbers are typically 12-digit numbers
+	taxCodeMYNumeric = regexp.MustCompile(`^\d{12}$`)
+
+	// Alphanumeric SST/Service Tax IDs — e.g., SST1234567890 or W10-12345678-123
+	taxCodeSST    = regexp.MustCompile(`^SST\d{10,12}$`)
```

Comment:
I'd ditch the `SST` part and just focus on the numbers. This could be normalized out. I think it'd also be important to detail why there are different formats, even if just in the comments.